contracts/src/v0.8/functions/dev/1_0_0/RouterBase.sol

```diff
@@ -270,44 +253,28 @@ abstract contract RouterBase is IRouterBase, Pausable, ITypeAndVersion, Confirme
     s_timelockProposal = TimeLockProposal({
       from: s_timelockBlocks,
       to: blocks,
-      timelockEndBlock: uint224(block.number + s_timelockBlocks)
+      timelockEndBlock: uint64(block.number + s_timelockBlocks)
     });
   }
 
-  /**
-   * @inheritdoc IRouterBase
-   */
+  // @inheritdoc IRouterBase
   function updateTimelockBlocks() external override onlyOwner {
     if (block.number < s_timelockProposal.timelockEndBlock) {
       revert TimelockInEffect();
     }
-    if (s_timelockBlocks == s_timelockProposal.to) {
-      revert AlreadyApplied();
-    }
     s_timelockBlocks = s_timelockProposal.to;
   }
 
   // ================================================================
   // |                     Pausable methods                         |
   // ================================================================
 
-  /**
-   * @inheritdoc IRouterBase
-   */
-  function isPaused() external view override returns (bool) {
-    return Pausable.paused();
-  }
-
-  /**
-   * @inheritdoc IRouterBase
-   */
+  // @inheritdoc IRouterBase
```

Comment:
Do we still have a way to verify if a contract is paused after dropping the `isPaused` method?

Reply:
Yes. OZ's `Pausable` gives us a public method for this that we inherit